docs/configuration.md

```diff
@@ -699,6 +699,48 @@ Advanced options example:
 
 You can define a `customization:` section in the config to configure some aspects of the look and feel of Turnilo.
 
+You can define custom share options section (`shareOptions:`), which will force specific formatting on exported files. It will ovewrite two default CSV and TSV exports in the share menu. You can define specific properties for number, time and time range types in the `locale:` section. Locale and Locale Options sections are optional but they will be set to default values if not provided.
+
+```yaml
+  shareOptions:
+
+  - title: "excel friendly export to TSV" # name displayed in the share menu
+    format: # "tsv" | "csv"
+    separator: # "\t" | "," | ... 
```

Comment:
What is the reason to define format AND separator? Format defines separator implicitly.